### PR TITLE
IA-3060: fixed issue with sticky session cookie and terraform applies

### PR DESCRIPTION
**Pull request by jsmigovsky**

### CODE REVIEW

**Review by bushong1 — APPROVED** [review]
lgtm
LGTM

**Review by mattjorg — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v9.3.1
+* **UPDATE**
+  * Added the `alb_sticky_cookie_type` variable with default of `lb_cookie` for backwards compatibiliy.
```
Comment (yaythecloud) [inline CHANGELOG.md:5]: compatibility

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+*  **FIX**
+  * Problem - When transitioning from an `app_cookie` to an `lb_cookie` AWS seems persist the `cookie_name` which is causing Terraform to detect a change on every plan evaluation even though the `cookie_name` is irrelevant.
```
Comment (yaythecloud) [inline CHANGELOG.md:8]: evaluation*

File: README.md [hunk 3]
```diff
@@ -94,7 +94,8 @@ Features:
 * `listeners` - The ALB listener port configuration. By default port 80 will be forwarded unless a certificate is provided then port 80 will redirect to port 443 which will then be forwarded. Here are some [examples](./examples/listener/main.tf) of listener configurations.
 * `ipv6` - Boolean to enable ipv6 on the ALB and Route53.  Ensure your VPC is configured to be ipv6 compatible before enabling.  Defaults to `false`.
 * `alb_sticky_duration` - By default, sticky sessions are disabled. Once a number value is provided, sticky sessions are enabled, and the provided number is used to determine sticky session's duration in seconds.
-* `alb_sticky_cookie_name` - By default a cookie type of lb_cookie will be used. Once assigned a string value a cookie type of app_cookie will be used. To enable sticky sessions please refer to variable alb_sticky_duration.
+* `alb_stucky_cookie_type` - By default a cookie type of `lb_cookie` will be used. Only `lb_cookie` and `app_cookie` are supported.
```
Comment (yaythecloud) [inline README.md:97]: sticky*

Comment (mshiyk) [inline README.md:97]: misspelled alb_stucky_cookie_type.
correction: alb_sticky_cookie_type

**Review by yaythecloud — APPROVED** [review]
Tested and approved.

**Pull request merged**